fire_services/fire_services-dao/src/test/java/project_alias/vehicles/VehicleTypeTest.java

```diff
@@ -39,12 +39,12 @@
      */
     @Test
     public void status_can_be_created_and_saved() {
-    	final var status = co(VehicleType.class).new_();

```

Comment:
Why this test is called `status_can_be_created_and_saved` if you are testing entity `Vehicle Type`?